src/adapters/poi_popup.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
-import renderStaticReact from 'src/libs/renderStaticReact';
+import ReactDOM from 'react-dom';
 import { Popup } from 'mapbox-gl--ENV';
 import ApiPoi from './poi/idunn_poi';
 import { isMobileDevice } from 'src/libs/device';
 import ReactPoiPopup from 'src/components/PoiPopup';
-import { listen } from 'src/libs/customEvents';
+import { fire, listen } from 'src/libs/customEvents';
 
 const WAIT_BEFORE_DISPLAY = 350;
+const WAIT_BEFORE_CLOSE = 200;
```

Comment:
Isn't 200ms a bit short to move from the marker to the popup content in some conditions ?

Reply:
@xem used 1 second at first, which was a long time to make the popup disappear if the user just left the marker without hovering the popup.
But maybe I got too strong with 200ms. What would you recommend? 500ms? Or maybe we can use the same value as the previous line.

Reply:
Ok, let's use 350ms as a precaution.